<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKInputDataPreset</class>
 <widget class="QWidget" name="SAKInputDataPreset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preset data editing</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="outportPushButton">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="deletePushButton">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="addPushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="infoLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="importPushButton">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QListWidget" name="itemListWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SAKInputDataPreset</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
